patches/chrome-browser-shell_integration.h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/shell_integration.h b/chrome/browser/shell_integration.h
+index 29e98f5b2839a44a488d6baec756b660935302dd..ea749b1be4e351bd5c38d23fd3e81e5db25e5c6f 100644
+--- a/chrome/browser/shell_integration.h
++++ b/chrome/browser/shell_integration.h
+@@ -236,6 +236,7 @@ class DefaultBrowserWorker : public DefaultWebClientWorker {
+   ~DefaultBrowserWorker() override;
+ 
+  private:
++   BRAVE_DEFAULT_BROWSER_WORKER_H
```

Comment:
Moved macro before `private:`.
It's hard to avoid using patching because `DefaultBrowserWorker` class doesn't have any its specific
symbols. Due to this, can't find anything for redefining. `worker_name_` is in another class. can't use it.